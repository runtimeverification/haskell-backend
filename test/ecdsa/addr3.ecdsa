#unparseData(#addrFromPrivateKey("0x0000000000000000000000000000000000000000000000000000000000000002"), 20)
