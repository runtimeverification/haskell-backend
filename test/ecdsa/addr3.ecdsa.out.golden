<k>
  "0x2b5ad5c4795c026514f8317c7a215e218dccd6cf" ~> .K
</k>
